locales/tr-TR/translation.json

```diff
@@ -0,0 +1,280 @@
+  "New wallet name": "Yeni cüzdan ismi",
+  "New password": "Yeni şifre",
+  "Retype password": "Şifreyi tekrar yazın",
+  "Passwords are different": "Şifre uyuşmazlığı",
+  "Make sure to keep your password secured as it cannot be changed in the future.": "Şifrenizi güvenli şekilde sakladığınızdan emin olunuz, daha sonrasında değiştirilemeyeceğini bilin.",
+  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "Alephium projesinin herhangi bir şekilde cüzdanınıza erişme şansı yoktur. Cüzdanınız yalnızca sizin sahipliğinizdedir.",
+  "Back": "Geri",
+  "Type your recovery phrase": "Kurtarma sözcüklerinizi yazın",
+  "Error while importing wallet": "Cüzdan içe aktarılırken hata oluştu",
+  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Kurtarma sözcüklerini güvenli bir yere sakladığınızdan emin olun! Cüzdanınıza özel gizli kurtarma sözcükleridir.",
+  "All good? Let's continue!": "Herşey yolunda mı ? Haydi devam edelim!",
+  "Everything is ready!": "Herşey hazır!",
+  "Welcome to Alephium.": "Alephium' a hoşgeldiniz.",
+  "Let's go!": "Haydi başlayalım!",
+  "welcomeScreenOneAddressPerGroupMessage": "Gelişmiş kullanıcı: <1>her bir gruba madencilik ya da defi için bir adres</1> ile başlamak ister misiniz ?",
+  "Show advanced options": "Gelişmiş seçenekleri göster",
+  "Hide advanced options": "Gelişmiş seçenekleri gizle",
+  "For mining or DeFi use.": "Madencilik ya da DeFi kullanımı içindir.",
+  "Address copied to clipboard!": "Adres panoya kopyalandı!",
+  "Your Wallet": "Cüzdanınız",
+  "Your address": "Adresiniz",
+  "I've copied the words, continue": "Sözcükleri kopyaladım, devam",
+  "Advanced operations reserved to more experienced users. A \"normal\" user should not need to use them very often, if not at all.": "Gelişmiş işlemler tecrübeli kullanıcılar için ayrılmıştır. \"Normal\" kullanıcının sıklıkla onları ihtiyaç duyulmaması halinde kullanımına gerek yoktur.",
+  "It appears that your wallet has too many UTXOs to be able to send this transaction. Please, consolidate (merge) your UTXOs first. This will cost a small fee.": "Görünüyor ki cüzdanınızda işlemin gönderilmesi için çok fazla UTXO mevcut. Lütfen öncelikle UTXO işlemlerini birleştiriniz. Bu çok az bir işlem ücretine mal olacaktır.",
+  "Welcome!": "Hoşgeldiniz!",
+  "Welcome back!": "Tekrar hoşgeldiniz!",
+  "Please choose a wallet and enter your password to continue.": "Lütfen devam etmek için cüzdanınızı seçip şifrenizi girin.",
+  "Create / import a new wallet": "Yeni cüzdan oluştur / içe aktar",
+  "(current)": "(şu anki)",
+  "Default address for sending transactions.": "İşlemlerin gönderileceği varsayılan adres.",
+  "Error while fetching data for address {{ hash }}": "Adres {{ hash }} için veri işlenirken hata oluştu",
+  "Error while fetching pending transactions for address {{ hash }}": "Adres {{ hash }} için bekleyen işlemler işlenirken hata oluştu",
```

Comment:
"Adres {{ hash }} için bekleyen işlemler işlenirken hata oluştu",
"Adres  için bekleyen işlemler işlenirken hata oluştu {{ hash }}",

belki {{ hash }} sonuna eklenebilir. hatalı işlemin hashini gösteriyorsa eğer.